### PR TITLE
MIMIR-793: Debugging of components coming and going

### PR DESCRIPTION
**Pull request by itemodahl**
Add logging of stickymenu render result length
Remove unnecessery setting of id
Change id of footer component to reactfooter to check if it is something else with the same id
Remove a required options on menubox alt description since it is making warnings

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ name: "Test Environment"
 trigger:
   branch:
     - master
+    - MIMIR-793-debuggingComponentsDisapearing
```
Comment (Garlov) [inline .drone.yml:25]: remove branch trigger for drone

File: src/main/resources/site/pages/default/default.es6 [hunk 2]
```diff
@@ -197,6 +197,21 @@ exports.get = function(req) {
...
+    items: breadcrumbs
+  })
+    .setId('breadcrumbs')
+    .uniqueId()
+
+  /*pageContributions = breadcrumbComponent.renderPageContributions({
+    pageContributions
+  })*/
+
+  log.info('Add id %s', JSON.stringify(breadcrumbComponent.renderBody(), null, 2))
```
Comment (Garlov) [inline src/main/resources/site/pages/default/default.es6:213]: remove log and comment

File: src/main/resources/site/pages/default/default.es6 [hunk 3]
```diff
@@ -217,40 +232,42 @@ exports.get = function(req) {
...
+    addAlerts(alerts, bodyWithBreadCrumbs, pageContributions) :
+    { body: bodyWithBreadCrumbs, pageContributions }
+
+  return {
+    body: bodyWithAlerts.body,
+    pageContributions: bodyWithAlerts.pageContributions
+  }
+}
+
+function addAlerts(alerts, body, pageContributions ){
```
Comment (Garlov) [inline src/main/resources/site/pages/default/default.es6:257]: lint

File: src/main/resources/site/pages/default/default.es6 [hunk 4]
```diff
@@ -217,40 +232,42 @@ exports.get = function(req) {
...
-  if (alerts.length > 0) {
-    const alertComponent = new React4xp('Alerts')
-      .setProps({
-        alerts
-      })
-      .setId('alerts')
-    body = alertComponent.renderBody({
+  const bodyWithAlerts = alerts.length ?
+    addAlerts(alerts, bodyWithBreadCrumbs, pageContributions) :
+    { body: bodyWithBreadCrumbs, pageContributions }
```
Comment (Garlov) [inline src/main/resources/site/pages/default/default.es6:249]: lint

**Review by Garlov — CHANGES_REQUESTED** [review]

File: .drone.yml (hunk 1 above)
Comment (itemodahl) [reply .drone.yml]: removed

File: src/main/resources/site/pages/default/default.es6 (hunk 3 above)
Comment (itemodahl) [reply src/main/resources/site/pages/default/default.es6]: linted

File: src/main/resources/site/pages/default/default.es6 (hunk 4 above)
Comment (itemodahl) [reply src/main/resources/site/pages/default/default.es6]: linted

**Review by Garlov — APPROVED** [review]

**Pull request merged**